cmd/noms/diff/diff_test.go

```diff
@@ -311,11 +346,217 @@ func TestNomsRefDiff(t *testing.T) {
 
 	tf := func(leftRight bool) {
 		buf := &bytes.Buffer{}
-		Diff(buf, r1, r2, leftRight)
+		PrintDiff(buf, r1, r2, leftRight)
 
 		test.EqualsIgnoreHashes(t, expected, buf.String())
 	}
 
 	tf(true)
 	tf(false)
 }
+
+func TestNomsDiffMap(t *testing.T) {
```

Comment:
Wondering if you can't combine these print vs. non-print versions rather than duplicating the creation and expected stuff?